sdk/cosmos/azure-cosmos-examples/src/main/java/com/azure/cosmos/examples/ChangeFeed/SampleChangeFeedProcessor.java

```diff
@@ -121,6 +121,7 @@ public static CosmosAsyncClient getCosmosClient() {
                 .key(SampleConfigurations.MASTER_KEY)
                 .connectionPolicy(ConnectionPolicy.getDefaultPolicy())
                 .consistencyLevel(ConsistencyLevel.EVENTUAL)
+                .returnMinimalResponse(false)
```

Comment:
ChangeFeedProcessor is a read-only, how minimal response impacts it?

Reply:
This is SampleChangeFeedProcessor example, which creates some documents before listening to them, so used it. This is not in the source code.